src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -172,6 +172,27 @@ protected def module_filter_basis : module_filter_basis 𝕜 E :=
   smul_left' := p.basis_sets_smul_left,
   smul_right' := p.basis_sets_smul_right }
 
+lemma filter_eq_infi (p : seminorm_family 𝕜 E ι) :
+  p.module_filter_basis.to_filter_basis.filter = ⨅ i, (𝓝 0).comap (p i) :=
+begin
+  refine le_antisymm (le_infi $ λ i, _) _,
+  { rw p.module_filter_basis.to_filter_basis.has_basis.le_basis_iff
+      (metric.nhds_basis_ball.comap _),
+    intros ε hε,
+    refine ⟨(p i).ball 0 ε, _, _⟩,
+    { convert p.basis_sets_mem {i} hε,
+      rw finset.sup_singleton },
```

Comment:
I slightly prefer:
```suggestion
    { rw ← (finset.sup_singleton : _ = p i),
      exact p.basis_sets_mem {i} hε, },
```
(since it avoids `convert`).